x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -195,6 +195,7 @@ STRING
     | '"'   ('\\' [btnfr"'\\] | ~[\r\n"\\])* '"'
     | '?"'  ('\\"' |~["\r\n])* '"'
     | '?\'' ('\\\'' |~['\r\n])* '\''
+    | '"""' ('\\\'' |~['\r\n])* '"""'
```

Comment:
Looks like `'` needs to be escaped under this format.
I also have some questions about the desired behavior of triple double quotes:

How would each of these be interpreted? I think we should know our edge cases up front. I'm wondering what the implication is for `"` at the beginning or at the end of a string. And how many of them are supported
* `""""""`
* `"""""x"""`
* `"""x"""""`
* `""""""""`

Reply:
> Looks like `'` needs to be escaped under this format.

Correct, I rushed and overlooked that.

> I also have some questions about the desired behavior of triple double quotes:
>
> How would each of these be interpreted? I think we should know our edge cases up front. I'm wondering what the implication is for `"` at the beginning or at the end of a string. And how many of them are supported
> 
> * `""""""`
> * `"""""x"""`
> * `"""x"""""`
> * `""""""""`

The extra `"` in the beginning or end of the string would be part of the final string:
* `` -> empty string
* `""x`
* `x""`
* `""`